src/components/TitleBar/styles/ViewTitleBar.module.css

```diff
@@ -55,10 +55,14 @@
     color: #888;
 }
 
-.moreButton {
+.moreText {
     margin-left: 5px;
 }
 
+.moreButtonSmall {
+    display: none !important;
```

Comment:
Can `!important` be avoided by increasing the specificity with `.strip .moreButtonSmall`?

Reply:
Done